src/pages/VestingClaim.tsx

```diff
@@ -120,17 +119,17 @@ function VestingClaim(): ReactElement {
 
   return (
     <>
-      <div className={styles.container}>
-        <div className={styles.gradient}></div>
-        <div className={styles.logoWrapper}>
-          <div className={styles.logo}>
+      <div>
+        <div></div>
+        <div>
+          <div>
             <img src={logo} />
           </div>
         </div>
-        <div className={styles.mainContent}>
+        <div>
```

Comment:
can you dbl check if you can get rid of this div too?

Reply:
Oh, Sorry, actually, I didn't finish the vesting claim page.
Thank you for letting me know. 👍🏻